### ISSUE TITLE
rstudio-server

### ISSUE THREAD

**Issue by bshor** [mwt/rstudio-apt-repo#3]
Your repo works perfectly for rstudio (desktop). But rstudio-server isn't on there, I think?

**Comment by mwt** [comment]
That's right. I just pushed a commit that should add it. We should find out it it worked in the next six hours or so.

**Comment by bshor** [comment]
It worked like a charm! :)
